client/src/app/primitives/modal/Modal.less

```diff
@@ -1,4 +1,16 @@
 :local(.ModalOverlay) {
+  --blue-base: #4d90ff;
```

Comment:
Can we make these declarations global => Ensure that they are being used independent from where the `.btn-xxx` is being used?